internal/app/admin/http/handler.go

```diff
@@ -165,6 +167,15 @@ func getParam(path string, prefix string) string {
 	return param
 }
 
+func getBoolQueryValue(values url.Values, key string) (bool, error) {
+	verbose := values.Get(key)
+	isVerbose, err := strconv.ParseBool(verbose)
+	if err == nil {
+		return isVerbose, nil
+	}
+	return false, err
```

Comment:
Can we remove the `verbose` from here too? This can be shortened to the following since Go defaults for bool will be false:
```
	val := values.Get(key)
	return strconv.ParseBool(val)
```